packages/paste-core/core-bundle/code-block/package.json

```diff
@@ -5,4 +5,4 @@
   "sideEffects": false,
   "main": "../dist/code-block.js",
   "types": "../dist/code-block.d.ts"
-}
+}
```

Comment:
Question: why is this in the PR?

Reply:
Unfortunately, if you build main, this is an uncommitted change. I think it slipped through somehow.